README.md

```
@@ -16,6 +16,7 @@ A powerful bot designed to enhance your server experience.<br/>
 [![][github-releasedate-shield]][github-releasedate-link]<br/>
 [![][discord-shield]][discord-link]
 [![][codecov-shield]][codecov-link]
+[![][codspeed-shield]][codspeed-link]
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)